decidim-core/app/mailers/decidim/user_update_mailer.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module Decidim
+  class UserUpdateMailer < ApplicationMailer
+    def notify(user, updates)
+      with_user(user) do
+        @user = user
+        @organization = user.organization
+        @updates = format_array(updates)
+        mail(to: user.email, subject: I18n.t(
+          "decidim.user_update_mailer.subject"
+        ))
+      end
+    end
+
+    private
+
+    def format_array(updates)
+      case updates.length
+      when 1
+        updates.first
+      when 2
+        "#{updates.first} and #{updates.last}"
+      else
+        "#{updates[0..-2].join(", ")}, and #{updates.last}"
+      end
+    end
```

Comment:
```suggestion
    def format_array(updates)
      case updates.length
      when 1
        I18n.t("list_of_updates",update: updates.first)
      else
        last = updates.last 
        I18n.t("list_of_updates",updates: updates.join(","), last_updated_field: last, count: updates.length + 1)
      end
    end
```

Reply:
Thanks for the suggest,
I've already made the changes with a similar strategy